pymc3/sampling.py

```diff
@@ -81,20 +82,20 @@ def instantiate_steppers(model, steps, selected_steps, step_kwargs=None):
 
     Parameters
     ----------
-    model : Model object
+    model: Model object
         A fully-specified model object
-    step : step function or vector of step functions
+    steps: step function or vector of step functions
```

Comment:
While `steps` is probably a more appropriate name, changing it now with break things. Do we think its worth it?

Reply:
Actually, I didn't change the argument name, I just brought the name of the argument in the docstring into agreement with the name in the function header.

Reply:
Sorry, misread that.